bin/logstash

```diff
@@ -46,4 +46,11 @@ fi
 . "$(cd `dirname $SOURCEPATH`/..; pwd)/bin/logstash.lib.sh"
 setup
 
-ruby_exec "${LOGSTASH_HOME}/lib/bootstrap/environment.rb" "logstash/runner.rb" "$@"
+if [ "$1" = "-V" ] || [ "$1" = "--version" ];
+then
+  LOGSTASH_VERSION_FILE="${LOGSTASH_HOME}/logstash-core/lib/logstash/version.rb"
+  LOGSTASH_VERSION=$(awk '/./{line=$0} END{print line}' $LOGSTASH_VERSION_FILE | cut -d '"' -f 2)
```

Comment:
ok, then no problem :-P

On Thu, Jun 16, 2016 at 1:11 PM João Duarte <notifications@github.com>
wrote:

> In bin/logstash
> <https://github.com/elastic/logstash/pull/5490#discussion_r67326823>:
>
> > @@ -46,4 +46,11 @@ fi
> >  . "$(cd `dirname $SOURCEPATH`/..; pwd)/bin/logstash.lib.sh"
> >  setup
> >
> > -ruby_exec "${LOGSTASH_HOME}/lib/bootstrap/environment.rb" "logstash/runner.rb" "$@"
> > +if [ "$1" = "-V" ] || [ "$1" = "--version" ];
> > +then
> > +  LOGSTASH_VERSION_FILE="${LOGSTASH_HOME}/logstash-core/lib/logstash/version.rb"
> > +  LOGSTASH_VERSION=$(awk '/./{line=$0} END{print line}' $LOGSTASH_VERSION_FILE | cut -d '"' -f 2)
>
> as I said, we are already using awk and sed on this script, so I used them
> too.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/elastic/logstash/pull/5490/files/e57c6542fd9cbaefc37f65c0e38013c33b3f822f#r67326823>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAELvEaaWNcOomAXayA9uNammvv-Lx6eks5qMS88gaJpZM4I1ZOL>
> .
>